docs/HOWTO.md

```diff
@@ -1,45 +1,34 @@
-How to create and track horizontal issues
-=========================================
+For Working Groups
+==================
 
-# What is a horizontal issue?
+* Apply the *-tracker label in your own repository to draw a horiztontal review group's attention to an issue in one of your own repositories.  Horizontal review groups may also apply the label if they are interested in tracking a particular issue.
 
-Horizontal issues are issues on specifications raised by an [horizontal group](https://www.w3.org/wiki/DocumentReview#How_to_get_horizontal_review).
+* Horizontal review groups may apply the *-needs-resolution to issues they expect to be resolved before the specification moves to a new maturity level.  
```

Comment:
Working Groups must not remove or add the *-needs-resolution label.

Reply:
I'm fine with adding that.

Reply:
added